### PR TITLE
ES6 syntax refactor

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   "description": "An open source blockchain explorer for monitoring Bodhi platform",
   "main": "index.js",
   "scripts": {
-    "start": "babel-node src/index.js",
-    "start:debug": "babel-node --inspect src/index.js",
-    "build": "babel -d build src",
-    "test": "mocha --compilers js:babel-core/register --recursive src/test/.",
+    "start": "node src/index.js",
+    "start:debug": "node --inspect src/index.js",
+    "build": "",
```
Comment (frankobe) [inline package.json]: remove if not use

File: package.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
+    "test": "mocha --recursive src/test/.",
     "lint:fix": "eslint --fix -- src/.",
     "pkg:mac": " cp qtum/mac/bin/qtumd bin/mac/qtumd ; npm run build ; pkg . -t node9-mac -o bin/mac/bodhi-app-mac",
```
Comment (frankobe) [inline package.json]: remove `npm run build`

File: package.json [hunk 3]
```diff
@@ -4,10 +4,10 @@
...
     "pkg:win64": " cp qtum/win64/bin/qtumd.exe bin/win64/qtumd.exe ; npm run build ; pkg . -t node9-win -o bin/win64/bodhi-app-win64.exe"
```
Comment (frankobe) [inline package.json]: remove `npm run build`

File: package.json [hunk 4]
```diff
@@ -39,20 +41,14 @@
     "node-fetch": "^1.7.3",
     "opn": "^5.2.0",
     "pg": "^7.4.0",
-    "qweb3": "^0.4.0",
+    "qweb3": "file:../qweb3.js/qweb3-0.6.0.tgz",
```
Comment (frankobe) [inline package.json]: from npm

File: src/index.js [hunk 5]
```diff
@@ -72,14 +69,15 @@ const openBrowser = async () => {
...
+  } catch (err) {
     console.debug('Chrome not found. Launching default browser.');
     await opn(`http://localhost:${PORT}`);
   }
 };
 
 // avoid using path.join for pkg to pack qtumd
-const qtumdPath = path.dirname(process.argv[0])+'/qtumd';
+const qtumdPath = `${path.dirname(__dirname)}/qtumd`;
+console.log(qtumdPath);
```
Comment (frankobe) [inline src/index.js]: remove

Comment (dwalintukan) [reply src/index.js]: removed

File: package.json (hunk 4 above)
Comment (dwalintukan) [reply package.json]: using remote module

File: package.json (hunk 1 above)
Comment (dwalintukan) [reply package.json]: removed

File: package.json (hunk 2 above)
Comment (dwalintukan) [reply package.json]: removed

File: package.json (hunk 3 above)
Comment (dwalintukan) [reply package.json]: removed

**Pull request merged**